src/Builders/FilterBuilder.php

```diff
@@ -125,6 +125,46 @@ public function when($value, callable $callback, callable $default = null)
         return $this;
     }
 
+    /**
+     * https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-distance-query.html
+     * @param string $field
+     * @param string $value
+     * @param array|string $distance
```

Comment:
According to the docs it can be only a string "1km" or "2mi", it can't be an array.

Reply:
Sorry, copy/paste problem :D